Content.Shared/Devour/Components/DevourerComponent.cs

```diff
@@ -59,6 +59,12 @@ public sealed partial class DevourerComponent : Component
         }
     };
 
+    [DataField]
+    public EntityWhitelist StorageWhitelist = new()
+    {
+        Components = new [] {"HumanoidAppearance"}
+    };
+
```

Comment:
I feel defaulting this is really weird. It should follow the other whitelists and be nulllable.